source/includes/steps-install-bi-connector.yaml

```diff
@@ -125,22 +112,37 @@ post: |
 
   You can examine the generated ``schema.drdl`` file, and remove any fields or
   tables that are not part of your intended schema.
+
 ---
-title: Start ``mongosqld``
+title: Edit your |bi| configuration file.
 ref: start-mongosqld
 level: 4
+content: |
+  Using your preferred text editor, create a |bi| configuration file.
+  To review the configuration file options, see :ref:`Configuration
+  File <config-format>`.
+
+---
+title: Start ``mongosqld`` as a system service.
+ref: start-mongosqld-service
+level: 4
 action:
-  - heading: "Linux / macOS"
+  - heading: "Linux / macOS using a configuration file"
     language: sh
     copyable: true
     code: |
-      mongosqld --schema schema.drdl --mongo-uri {your.mongohost.com}
-  - heading: "Windows"
+      mongosqld install --config {pathToConfigFile}/mongosqld.conf
+      service mongosql start
+  - heading: "Windows using a configuration file"
     language: powershell
     copyable: true
     code: |
-      "C:\Program Files\MongoDB BI Connector\bin\mongosqld.exe" --schema schema.drdl --mongo-uri {your.mongohost.com}
+      "C:\Program Files\MongoDB BI Connector\bin\mongosqld.exe" install --config {pathToConfigFile}\mongosqld.conf
```

Comment:
Should this be the default install path on windows? I'm not sure what that is off the top of my head, but it isn't this.